crates/wasm-encoder/src/elements.rs

```diff
@@ -113,7 +118,7 @@ impl ElementSection {
                 table: None,
                 offset,
             } => {
-                self.bytes.extend(encoders::u32(0x00 | expr_bit));
+                self.bytes.extend(encoders::u32(/* 0x00 | */ expr_bit));
```

Comment:
I realize that clippy is the one telling you to do something about this, but personally I do sort of like the symmetry of the cases here with the leading byte. I also don't want `#[allow(some_clippy_thing)]` though, so I suppose this is the next best thing.

Reply:
I'm happy to put it back and live with the warning too.